EasyPost/Shipment.cs

```diff
@@ -134,6 +134,29 @@ public void GetRates()
             rates = request.Execute<Shipment>().rates;
         }
 
+        /// <summary>
+        ///     Refresh the rates for this Shipment.
+        /// </summary>
+        /// <param name="parameters">Optional dictionary of parameters for the API request.</param>
+        /// <exception cref="ResourceAlreadyCreated">Shipment has not be created server-side yet.</exception>
+        public void RefreshRates(Dictionary<string, object> parameters = null)
```

Comment:
Let's call this `RegenerateRates`. We unfortunately do not have uniformity of names for this method across libraries, but the majority of them use `regenerate` so let's try to keep it as close as possible.

Reply:
Resolved in 71c3033